presto-main/src/main/java/com/facebook/presto/sql/relational/ConnectorRowExpressionService.java

```diff
@@ -54,9 +55,9 @@ public DomainTranslator getDomainTranslator()
     }
 
     @Override
-    public ExpressionOptimizer getExpressionOptimizer()
+    public ExpressionOptimizer getExpressionOptimizer(ConnectorSession session)
```

Comment:
It seems like this change is made in a later commit (https://github.com/prestodb/presto/pull/24144/commits/2d609d0aace7246a922029cf11ca16c72cc1f01b ), whereas the function signature in `RowExpressionService` was changed in prior commit. This will cause the build to fail.
Please make this change in a prior commit.